templates/stats/publications.tpl

```diff
@@ -172,6 +172,82 @@
 									{translate key="common.pagination.label"}
 								</a>
 							</div>
+							<pkp-button
+								ref="downloadReportModalButton"
+								@click="$modal.show('downloadReport')"
+							>
+								{translate key="common.downloadReport"}
+							</pkp-button>
+							<modal
+								close-label="Close"
```

Comment:
This needs to be localized. You can use `close-label="{translate key="common.close"}"` in template files.